addons/binding/org.openhab.binding.globalcache/ESH-INF/thing/gc-100-06.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="globalcache"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- GlobalCache GC-100-06 device contains one SL module, one IR module-->
+    <thing-type id="gc100_06">
+
+        <label>GlobalCache GC-100-06</label>
+        <description>GC-100-06 Infrared and Serial Device</description>
+
+        <channels>
+            <channel id="m1c1" typeId="channel-type-sl">
+                <label>Module 1, Connector 1</label>
```

Comment:
Groups are a good idea.  Makes the UI presentation much cleaner.  Thanks for that suggestion, @kaikreuzer